src/components/Graphs/index.jsx

```diff
@@ -29,12 +38,23 @@ const Graphs = ({ benchmarkData, overviewMode }) => (
     }
     {Object.keys(benchmarkData.graphs)
       .sort(overviewMode ? undefined : sortOverviewFirst)
-      .map(key => benchmarkData.graphs[key])
+      .map((key) => {
+        const graph = benchmarkData.graphs[key];
+        graph.anchor = key;
+        return graph;
+      })
       .map(({
-        chartJsData, chartJsOptions, jointUrl, title,
+        chartJsData, chartJsOptions, jointUrl, title, anchor,
       }) => (
         <div key={title}>
-          <h2>{title}</h2>
+          <h2>
+            <a id={anchor} href={`#${anchor}`} style={styles.anchorTag}>
+              <svg width="20" height="20" viewBox="0 0 24 24" style={styles.anchorIcon}>
+                <path fill="#000000" d="M5.41,21L6.12,17H2.12L2.47,15H6.47L7.53,9H3.53L3.88,7H7.88L8.59,3H10.59L9.88,7H15.88L16.59,3H18.59L17.88,7H21.88L21.53,9H17.53L16.47,15H20.47L20.12,17H16.12L15.41,21H13.41L14.12,17H8.12L7.41,21H5.41M9.53,9L8.47,15H14.47L15.53,9H9.53Z" />
+              </svg>
```

Comment:
Don't copy/paste the svg but install the package.

Look at this diff to see how to use the `mdi-react` package:
https://github.com/mozilla/firefox-code-coverage-frontend/commit/4a676c668fa7146105cf4ebc06c2e6d853c57531